_data-prepper/pipelines/configuration/sources/documentdb.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: documentdb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 2
+---
+
+# documentdb
+
+The `documentdb` source reads documents from [Amazon DocumentDB](https://aws.amazon.com/documentdb/) collections.
+It can read historical data from an export and keep up-to-date on the data using DocumentDB [Change Streams](https://docs.aws.amazon.com/documentdb/latest/developerguide/change_streams.html).
+
+The `documentdb` source will read data from DocumentDB and put that data into an [Amazon S3](https://aws.amazon.com/s3/) bucket.
+From there, other Data Prepper workers will read from the S3 bucket to process data.
+
+## Usage
+
+```
+version: "2"
+documentdb-pipeline:
+  source:
+    documentdb:
+      host: "docdb-mycluster.cluster-random.us-west-2.docdb.amazonaws.com"
+      port: 27017
+      authentication:
+        username: ${{aws_secrets:secret:username}}
+        password: ${{aws_secrets:secret:password}}
+      aws:
+        sts_role_arn: "arn:aws:iam::123456789012:role/MyRole"
+      s3_bucket: my-bucket
+      s3_region: us-west-2
+      collections:
+        - collection: my-collection
+          export: true
+          stream: true
+      acknowledgments: true
+```
```

Comment:
Should we add the copy button to this example? {% include copy-curl.html %}

Reply:
Yea, that is a great idea.

Reply:
I added this in my latest revision.